### DISCUSSION TITLE
Show Alias Name in the tile view on the aliases page

### DISCUSSION THREAD

**Discussion by gregforcey** [simple-login/app#424] [Feature Requests]
For each alias, there is a field called `Alias Name` where you can name each alias. Unfortunately `Alias Name` is not shown by default when viewing all the aliases in tile view on the aliases page. The only way to see it is to click on the more dropdown arrow and this has to be done individually for each alias. This limitation is particularly problematic for random aliases (by random words or UUID) because it is difficult to tell what the alias is for without clicking the more dropdown arrow. If the `alias name` was shown by default it would be much easier to determine, at a glance, what the alias is for. Ideally when sorting by aliases A-Z or Z-A, it would sort on this `alias name` as well. 

The only workaround that I've found is to just use a custom alias for each site with the site's name in the alias, as the alias itself is shown in the default view. But this doesn’t work if you want to use random words or UUID. It also doesn't work if you don't want to put the site's name in the alias for some reason. 

Thanks for considering.

**Comment by nguyenkims** [comment]
@gregforcey there's this feature request https://github.com/simple-login/app/discussions/427 that disables altogether the "more" button so all info is immediately shown. Feel free to upvote it to make it happen sooner :)

Please note that you might want to use "alias note" as "alias name" is the name that's used when you send an email from the alias.

**Reply by gregforcey** [reply]
Ok, I miised that feature request #427. Feel free to delete this one if you feel that it's a duplicate. 

I did not know about alias note being used when an e-mail is sent. Thanks for the info.

**Reply by nguyenkims** [reply]
No worries, I'll lock this thread then. Btw the alias **name** is used when sending emails from an alias and not **note** :)
